portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -55,26 +69,70 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             String name = (String) entity.getProperty("name");
             Date timestamp = (Date) entity.getProperty("timestamp");
             String text = (String) entity.getProperty("text");
+            String imageUrl = (String) entity.getProperty("imageUrl");
 
-            Comment newComment = new Comment(id, name, timestamp, text);
+            Comment newComment = new Comment(id, name, timestamp, text, imageUrl);
             comments.add(newComment);
         }
         
+        Map payload = new HashMap<String, Object>();
+        payload.put("commentsList", comments);
+        String uploadUrl = this.blobstore.createUploadUrl("/comments");
+        payload.put("uploadUrl", uploadUrl);
```

Comment:
I would serve this in different API since it has nothing to do with getting comments.